src/main/resources/reference.conf

```diff
@@ -35,3 +35,21 @@ nautilus.metering {
     keys: [ "" ]
   }
 }
+
+akka {
+  actor {
+    default-dispatcher {
+      type = "Dispatcher"
```

Comment:
can you change new config to be consistent and replace `=` with `:` as it is in the structure above?

Reply:
Done